app/store/ServicesList.js

```diff
@@ -10,7 +10,7 @@ Ext.define('Rambox.store.ServicesList', {
 
 	,proxy: {
 		type: 'ajax',
-		url: 'https://us-central1-rambox-d1326.cloudfunctions.net/ceApps',
+		url: 'https://raw.githubusercontent.com/saenzramiro/rambox/gh-pages/api/services.json',
```

Comment:
Why it does not use rambox app repository but someone else? I know that he is developer of Rambox CE, but still. Anytime, there is a risk that the repository/branch can be deleted.

It can use https://raw.githubusercontent.com/ramboxapp/community-edition/gh-pages/api/services.json